pkg/http/api.go

```diff
@@ -1815,7 +1793,7 @@ func (a *api) onGetActorState(reqCtx *fasthttp.RequestCtx) {
 		respond(reqCtx, withError(fasthttp.StatusInternalServerError, msg))
 		log.Debug(msg)
 	} else {
-		if resp == nil || len(resp.Data) == 0 {
+		if resp == nil || resp.Data == nil {
```

Comment:
Should not be reverted

```suggestion
		if resp == nil || len(resp.Data) == 0 {
```

Reply:
Btw this is the reason why the E2E tests are failing I think. We made that change because the tests were not working properly.